atlp-rwanda/destructors-ec-bn
# bg-fix-develop

**What does this PR do?**
- This PR adds **cors and product description** to the project
**Changes Made**
- Installed **cors** .
**Checklist**
- [x] Cors has been installed 
- [x] Products description has been added
**How this should be manually testing**
- Clone  this [repo](https://github.com/atlp-rwanda/destructors-ec-fe.git) 
- Got to the repo directory by  writing this `cd  destructors-ec-fe` in terminal
- Write this command in terminal `git checkout `bg-fix-develop`
- Run `npm install` 
- For running server `npm run dev`
- Check you frontend that fetch products from backend with products description
